quickassist/lookaside/access_layer/src/common/crypto/sym/include/lac_sym_key.h

```diff
@@ -72,15 +72,15 @@
  *
  * @lld_overview
  *
- * Key generation component is reponsible for SSL, TLS & MGF operations. All
+ * Key generation component is responsible for SSL, TLS & MGF operations. All
  * memory required for the keygen operations is got from the keygen cookie
  * structure which is carved up as required.
  *
  * For SSL the QAT accelerates the nested hash function with MD5 as the
  * outer hash and SHA1 as the inner hash.
  *
  * Refer to sections in draft-freier-ssl-version3-02.txt:
- *      6.1 Asymmetric cryptographic computations - This refers to coverting
+ *      6.1 Asymmetric cryptographic computations - This refers to converting
  *          the pre master secret to the master secret.
```

Comment:
Could the tool be taught to rewrite `pre master` as `pre-master`?

Reply:
I've hand fixed this and re-pushed the change